src/actions/AuthActions.js

```diff
@@ -104,7 +92,9 @@ export const setGuestToken = ({navigate}) => {
 
 export const logUserOutOfFacebook = ({navigate}) => {
     return async (dispatch) => {
-        await AsyncStorage.removeItem(tokens.USER_TOKEN);
+
+        // purge all async persisted state
+        await persistor.purge();
```

Comment:
Now when we log out, instead of removing the token manually from `AsyncStorage`, we can call `persistor.purge()` and `redux-persist` will reach in and clean out all of the persistent data from `AsyncStorage` that it was managing.